docs/articles/blog/2019-2-7-testcafe-v1-0-0-released.md

```diff
@@ -0,0 +1,162 @@
+
+Meet TestCafe v1.0.0, our first major update that includes some big features: video recording, configuration file, "live mode" for rapid test development and much more.
+
+<!--more-->
+
+## Breaking Changes
+
+TestCafe v1.0.0 introduces minor changes to the behavior and programming interface. You can find the details in the [migration guide](https://devexpress.github.io/testcafe/blog/migration-from-testcafe-v0-x-y-to-v1-0-0.html).
+
+## Enhancements
+
+### ⚙ Video Recording ([#2151](https://github.com/DevExpress/testcafe/issues/2151))
+
+You can now [record videos of test runs](https://devexpress.github.io/testcafe/documentation/using-testcafe/common-concepts/screenshots-and-videos.html#record-videos) in Google Chrome and Mozilla Firefox. To enable video recording, [install the FFmpeg library](https://devexpress.github.io/testcafe/documentation/using-testcafe/common-concepts/screenshots-and-videos.html#prerequisites) and then do one of the following:
+
+* specify the [--video](https://devexpress.github.io/testcafe/documentation/using-testcafe/command-line-interface.html#--video-basepath) command line flag,
+
+    ```sh
+    testcafe chrome test.js --video artifacts/videos/
+    ```
+
+* call the [runner.video](https://devexpress.github.io/testcafe/documentation/using-testcafe/programming-interface/runner.html#video) API method,
+
+    ```js
+    runner.video('artifacts/videos/');
+    ```
+
+* specify the [videoPath](https://devexpress.github.io/testcafe/documentation/using-testcafe/configuration-file.html#videopath) configuration file property (configuration file is also a new feature, see below).
+
+    ```json
+    {
+        "videoPath": "artifacts/videos/"
+    }
+    ```
+
+TestCafe records all tests and saves each recording in a separate file. You can change this behavior in [video options](https://devexpress.github.io/testcafe/documentation/using-testcafe/common-concepts/screenshots-and-videos.html#basic-video-options). You can also customize [video encoding parameters](https://devexpress.github.io/testcafe/documentation/using-testcafe/common-concepts/screenshots-and-videos.html#video-encoding-options).
+
+### ⚙ Configuration File ([#3131](https://github.com/DevExpress/testcafe/issues/3131))
+
+TestCafe now allows you to store its settings in the `.testcaferc.json` [configuration file](https://devexpress.github.io/testcafe/documentation/using-testcafe/configuration-file.html) (with support for `JSON5` syntax).
+
+```json
+{
+    "browsers": "chrome",
+    "src": ["/home/user/auth-tests/fixture-1.js", "/home/user/mobile-tests/"],
+    "reporter": {
+        "name": "xunit",
+        "output": "reports/report.xml"
+    },
+    "screenshotPath": "/home/user/tests/screenshots/",
+    "takeScreenshotsOnFails": true,
+    "videoPath": "/home/user/tests/videos/",
+    "pageLoadTimeout": 1000,
+    "hostname": "host.mycorp.com"
+    // and more
+}
+```
+
+Keep the configuration file in the project's root directory from which you run TestCafe.
```

Comment:
Why "Keep the configuration file in the project's root directory"? Where else would it be?

Reply:
I don't see a problem here - if you tell users that they should create a file, it's natural to say where to place it (how can you not say about it).